packages/frontend/hooks/src/use-block-suite-workspace-page.ts

```diff
@@ -55,8 +94,10 @@ export function useBlockSuiteWorkspacePage(
   const page = useAtomValue(pageAtom);
 
   useEffect(() => {
-    if (!page?.loaded) {
-      page?.waitForLoaded().catch(console.error);
+    if (page && !page.loaded) {
+      loadPage(page).catch(err => {
```

Comment:
I think you can use p-queue.add [priority](https://github.com/sindresorhus/p-queue#priority) option, jump into the queue and load the page here as quickly as possible, and reducing interference from automatic page loading.

Reply:
Great suggestion!